crates/sui-types/src/deny_list_v2.rs

```diff
@@ -141,37 +145,50 @@ pub fn check_coin_deny_list_v2_during_execution(
             .entry(coin_type.to_canonical_string(false))
             .or_insert_with(BTreeSet::new)
             .insert(owner);
-        num_regulated_transfers += 1;
+        num_non_gas_coin_owners += 1;
```

Comment:
btw if you really want to be accurate, you may need to do += 1 based on whether insert is successful, i.e. there may be the same owner showing up multiple times for the same coin.

Reply:
Refactored